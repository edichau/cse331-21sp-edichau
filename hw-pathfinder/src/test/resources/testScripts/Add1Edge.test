CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B

AddEdge simpleGraph A B 10.0