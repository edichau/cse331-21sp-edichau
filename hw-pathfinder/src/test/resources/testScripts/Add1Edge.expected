created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph

added edge 10.000 from A to B in simpleGraph